api/Start.cs

```diff
@@ -31,31 +32,39 @@ public static class Start
             [HttpTrigger(AuthorizationLevel.Function, "get", Route = "start")] HttpRequest req,
             ILogger log)
         {
-            ConfigurationModel config = configService.GetConfiguration();
-
-            try
-            {
-                AzureMediaServicesClient client = await authService.GetClientAsync();
+            using (SentrySdk.Init())
+            {
+                ConfigurationModel config = configService.GetConfiguration();
 
-                InputRequestService inputRequestService = new InputRequestService(client, config);
-                StartController startController = new StartController(client, config);
-
-                InputRequestModel inputModel = await inputRequestService.GetInputRequestModelAsync(req);
-                StatusChangeOutputModel outputModel = await startController.StartServicesAsync(inputModel);
+                try
+                {
+                    AzureMediaServicesClient client = await authService.GetClientAsync();
 
-                await WebhookService.CallWebhookAsync(config.WebhookStartSuccess, ActionEnum.Start, outputModel.Status.Summary);
-                return successResponseService.CreateResponse(outputModel, HttpStatusCode.Created);
-            }
-            catch (AppException e)
-            {
-                await WebhookService.CallWebhookAsync(config.WebhookStartFailure, ActionEnum.Start, ResourceStatusEnum.Error);
-                return errorResponseService.CreateResponse(e);
-            }
-            catch (Exception e)
-            {
-                await WebhookService.CallWebhookAsync(config.WebhookStartFailure, ActionEnum.Start, ResourceStatusEnum.Error);
-                return errorResponseService.CreateResponse(e);
+                    InputRequestService inputRequestService = new InputRequestService(client, config);
+                    StartController startController = new StartController(client, config);
+
+                    InputRequestModel inputModel = await inputRequestService.GetInputRequestModelAsync(req);
+                    StatusChangeOutputModel outputModel = await startController.StartServicesAsync(inputModel);
+
+                    await WebhookService.CallWebhookAsync(config.WebhookStartSuccess, ActionEnum.Start, outputModel.Status.Summary);
+                    return successResponseService.CreateResponse(outputModel, HttpStatusCode.Created);
+                }
+                catch (AppException e)
+                {
+                    return await ReportError(config, e);
+                }
+                catch (Exception e)
+                {
+                    return await ReportError(config, e);

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add '.ConfigureAwait(false)' to this call to allow execution to continue in any thread.](https://app.codacy.com/gh/literal-life-church/live-streaming-api/pullRequest?prid=5532272)